app/index.js

```diff
@@ -1,11 +1,12 @@
-'use strict';
-const App = require('app');
-const Path = require('path');
-const BrowserWindow = require('browser-window');
-const Tray = require('tray');
+"use strict";
+const App = require("app");
+const Path = require("path");
+const BrowserWindow = require("browser-window");
+const Tray = require("tray");
```

Comment:
what's the rationale for `" -> '`? The [Google JavaScript Style Guide](http://google-styleguide.googlecode.com/svn/trunk/javascriptguide.xml?showone=Strings#Strings) recommends `'`' for consistency.